packages/frontend-shared/src/styles/shared.scss

```diff
@@ -51,13 +51,4 @@ body {
 // Light scrollbars, anything with light:
 .force-light {
   color-scheme: light;
-}
-
-/**
- * Common theming
- * -------------------------
- */
-
-::selection {
-  @apply bg-gray-200 bg-opacity-30;
```

Comment:
This ::selection class is super broad and is trying to match over thousands of elements when running tests - with zero matches. I don't see any impact to removing it. These ms during recalculatestyles will be removed.

![Screenshot 2025-07-09 at 10 55 59 AM](https://github.com/user-attachments/assets/9257343e-d34c-4938-801e-e0a82335d6e7)